spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe CommentsController, type: :controller do
+  describe "POST #create" do
+    before do
+      @project = FactoryGirl.create(:project)
+      sign_in(@project.user)
+    end
+    it 'Response code is 200' do
+      xhr :post, :create, comment: { polycomment_type: "project", polycomment_id: 3, issue: false, project_name: @project.name, body: "heads are cool" }
```

Comment:
> what do you think about this: https://github.com/jackdempsey/acts_as_commentable

I don't think we need it, but if you want to use it - go ahead.

> Should I make keep submitting changes to current PR or should I start a new one?

Your choice :)